### ISSUE TITLE
SRW - implement calculation of diffraction angle for crystals

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#261]
Diffraction angle for crystals can be calculated automatically using d-spacing value from Stepanov's x-server http://x-server.gmca.aps.anl.gov/cgi/X0h_form.exe?xway=2&wave=20.368&coway=0&code=Silicon&i1=1&i2=1&i3=1&df1df2=-1&modeout=1.

**Comment by mrakitin** [comment]
The angle is already calculated:
```
 Bragg angle (degr.):                 QB=  5.5704    
 Interplanar spacing (A):              d=  3.1355 
```

**Comment by mrakitin** [comment]
Implemented, however the angle does not cause any changes in the results for the intensity after crystals. Need to discuss with Oleg, there may be a problem in SRW. There is nothing else to do on Sirepo side.

**Comment by mrakitin** [comment]
Will discuss the possible issue within https://github.com/mrakitin/SRW/issues/4.

**Comment by mrakitin** [comment]
Need to also get d-spacing from the server for future usage:
```
Interplanar spacing (A):              d=  3.1355
```

**Comment by mrakitin** [comment]
Fixed in f56edaa.
